README.md

```diff
@@ -2,25 +2,39 @@
 
 [![GitHub license](https://img.shields.io/github/license/PX4/px4_msgs.svg)](https://github.com/PX4/px4_msg/blob/master/LICENSE) [![Build package](https://github.com/PX4/px4_msgs/workflows/Build%20package/badge.svg)](https://github.com/PX4/px4_msgs/actions)
 
-This package contains the ROS2 message definitions of the [PX4 Pro ecosystem](https://px4.io/). Building this package generates all the required interfaces to interface ROS2 nodes with the PX4 autopilot internals, which use the [uORB messaging API](https://dev.px4.io/en/middleware/uorb.html). Currently the messages of this package represent a dependency to [`px4_ros_com` package](https://github.com/PX4/px4_ros_com).
+This interface package contains the ROS 2 message definitions of the [PX4 Pro ecosystem](https://px4.io/). Building this package generates all the required interfaces to interface ROS 2 nodes with the PX4 autopilot internals.
 
-## uORB message definitions
+## Supported versions and compatibility
 
-The uORB message definitions, which represent the counter-part of the ROS2 messages found in this package, can be found on the [PX4 Firmware repository](https://github.com/PX4/Firmware).
+Depending on the PX4 and ROS versions you want to use, you need to checkout the appropriate branch of this package:
 
-## How are these messsage definitions generated?
+| PX4 version   | ROS 2 versions          | branch                                                            |
```

Comment:
I think we should also add the Ubuntu version we support for each @beniaminopozzan. Would that be possible?

Reply:
Yes, done!

Reply:
Would it be possible to use this format?

```
| ROS     | PX4    | Ubuntu       |
| ------- | ------ | ------------ |
| Foxy    | [v1.13](https://github.com/PX4/px4_msgs/tree/release/1.13)  | Ubuntu 20.04 |
| Humble  | ≥ [v1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) | Ubuntu 22.04 |
| Rolling | ≥ [v1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) | Ubuntu 22.04 |
```

Reply:
This exclude the combination Foxy + v1.14, I can expand on this though:


| PX4            | ROS 2   | Ubuntu       | branch                                                            |
|----------------|---------|--------------|-------------------------------------------------------------------|
| [v1.13](https://github.com/PX4/px4_msgs/tree/release/1.13)           | Foxy    | Ubuntu 20.04 | [release/1.13](https://github.com/PX4/px4_msgs/tree/release/1.13) |
| [v1.14](https://github.com/PX4/px4_msgs/tree/release/1.14)           | Foxy    | Ubuntu 20.04 | [release/1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) |
| [v1.14](https://github.com/PX4/px4_msgs/tree/release/1.14)           | Humble  | Ubuntu 22.04 | [release/1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) |
| [v1.14](https://github.com/PX4/px4_msgs/tree/release/1.14)           | Rolling | Ubuntu 22.04 | [release/1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) |
| [main](https://github.com/PX4/px4_msgs/tree/main)  | Foxy    | Ubuntu 22.04 | [main](https://github.com/PX4/px4_msgs)                           |
|[main](https://github.com/PX4/px4_msgs/tree/main)  | Humble  | Ubuntu 22.04 | [main](https://github.com/PX4/px4_msgs)                           |
| [main](https://github.com/PX4/px4_msgs/tree/main)   | Rolling | Ubuntu 22.04 | [main](https://github.com/PX4/px4_msgs)                           |

What do you think?
I cannot put "≥ v1.14" as v1.14 (release/1.14) and `main` have already diverged in term off messages definitions.